src/renderer/containers/Settings/SupportBox.tsx

```diff
@@ -81,6 +83,9 @@ export const SupportBox = () => {
             Copy logs
           </Button>
         </div>
+        <div>
+          <Diagnostic />
```

Comment:
```suggestion
          <NetworkDiagnosticsButton />
```
`Diagnostic` is very generic. This is specific to network diagnostics so it should be renamed as such.